tests/x11/nis_client.pm

```diff
@@ -121,15 +121,15 @@ sub run {
     # we have to stop the firewall, see bsc#999873 and bsc#1083487#c36
     systemctl 'stop ' . $self->firewall;
 
-    mutex_lock('nis_ready');    # wait for NIS server setup
+    mutex_wait('nis_nfs_server_ready');    # wait for server setup
```

Comment:
I think that with this `mutex_wait` we will be trying to unlock twice, one in the server when the mutex is created and one here in the client, the result is the same of course but just to mention it.